### ISSUE TITLE
Treat missing alt attributes as Warning instead of Problem

### ISSUE THREAD

**Issue by adrien-delhorme** [kapt-labs/django-check-seo#48]
**Describe the bug**
An `<img>` missing an `alt` attribute is filed as Problem.

According to [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html), some images *must not* have `alt` attributes:
>  Sometimes there is non-text content that really is not meant to be seen or understood by the user. Transparent images used to move text over on a page; an invisible image that is used to track usage statistics; and a swirl in the corner that conveys no information but just fills up a blank space to create an aesthetic effect are all examples of this. Putting alternative text on such items just distracts people using screen readers from the content on the page. Not marking the content in any way, though, leaves users guessing what the non-text content is and what information they may have missed (even though they have not missed anything in reality). This type of non-text content, therefore, is marked or implemented in a way that assistive technologies (AT) will ignore it and not present anything to the user. 

**Expected behavior**
File missing `alt` attributes as warnings and explain why and how to fill `alt` attributes.
Quotes and links to WCAG (and the RGAA french equivalent) should be present.
